apps/emqx_bridge/src/schema/emqx_bridge_enterprise.erl

```diff
@@ -124,7 +127,16 @@ resource_type(oracle) -> emqx_oracle;
 resource_type(iotdb) -> emqx_bridge_iotdb_impl;
 resource_type(rabbitmq) -> emqx_bridge_rabbitmq_connector;
 resource_type(kinesis_producer) -> emqx_bridge_kinesis_impl_producer;
-resource_type(greptimedb) -> emqx_bridge_greptimedb_connector.
+resource_type(greptimedb) -> emqx_bridge_greptimedb_connector;
+resource_type(azure_event_hub_producer) -> emqx_bridge_kafka_impl_producer.
```

Comment:
Nit: without a comment, this looks like a typo 😅

Reply:
Well, it's pretty much the same as we already do with Postgres, Matrix and Timescale. :sweat_smile: 

I can add a comment to assert that.